src/sentry/api/endpoints/project_rules.py

```diff
@@ -73,23 +82,219 @@ def find_duplicate_rule(rule_data, project, rule_id=None):
+}
+```
+
+- The issue affects more than `value` percent of sessions in `interval`:
+```json
+{
+    "id": "sentry.rules.conditions.event_frequency.EventFrequencyPercentCondition",
+    "value": <Number>,
+    "interval": <"5m" OR "10m" OR "30m" OR "1h">
+}
+```
+""",
+    )
+    frequency = serializers.IntegerField(
+        min_value=5,
+        max_value=60 * 24 * 30,
+        help_text="How often the alert rule can be triggered for a particular issue, in seconds.",
+    )
+    name = GlobalParams.name("The name for the rule.")
+    environment = GlobalParams.ENVIRONMENT
+    filterMatch = serializers.ChoiceField(
+        choices=(("all", "all"), ("any", "any"), ("none", "none")),
+        required=False,
+        help_text="An operator determining which filters need to hold before any actions take place. Required when `filters` is passed in.",
+    )
+    filters = serializers.ListField(
+        child=RuleNodeField(type="filter/event"),
+        required=False,
+        help_text="""
+A list of filters that determine if a rule fires after the listed conditions have been met. See below for a list of possible filters.
+
+- The issue is `comparison_type` than `value` `time`:
+```json
+{
+    "id": "sentry.rules.filters.age_comparison.AgeComparisonFilter",
+    "comparison_type": <"older" OR "newer">,
+    "value": <Number>,
+    "time": <"minute" OR "hour" OR "day" OR "week">
+}
+```
+
+- The issue has happened at least `value` times (Note: this is approximate):
+```json
+{
+    "id": "sentry.rules.filters.issue_occurrences.IssueOccurrencesFilter",
+    "value": <Number>
+}
+```
+
+- The issue is assigned to No One:
+```json
+{
+    "id": "sentry.rules.filters.assigned_to.AssignedToFilter",
+    "targetType": "Unassigned"
+}
+```
+
+- The issue is assigned to `targetType`:
+```json
+{
+    "id": "sentry.rules.filters.assigned_to.AssignedToFilter",
+    "targetType": <"Team" OR "Member">,
+    "targetIdentifier": <Number>
+}
+```
+
+- The event is from the latest release:
+```json
+{
+    "id": "sentry.rules.filters.latest_release.LatestReleaseFilter"
+}
+```
+
+- The issue's category is equal to `value`:
+```json
+{
+    "id": "sentry.rules.filters.issue_category.IssueCategoryFilter",
+    "value": <1 OR 2 OR 3 OR 4 OR 5>
+}
+```
+
+- The event's `attribute` value `match` `value`:
+```json
+{
+    "id": "sentry.rules.conditions.event_attribute.EventAttributeCondition",
+    "attribute": <"message" OR "platform" OR "environment" OR "type" OR "error.handled" OR "error.unhandled" OR "error.main_thread" OR "exception.type" OR "exception.value" OR "user.id" OR "user.email" OR "user.username" OR "user.ip_address" OR "http.method" OR "http.url" OR "http.status_code" OR "sdk.name" OR "stacktrace.code" OR "stacktrace.module" OR "stacktrace.filename" OR "stacktrace.abs_path" OR "stacktrace.package" OR "unreal.crashtype" OR "app.in_foreground">
+    "match": <"co" OR "ew" OR "eq" OR "is" OR "nc" OR "new" OR "ne" OR "ns" OR "nsw" OR "sw">
```

Comment:
For things like this where the available values are abbreviated or not entirely clear as to what they mean, what would be the best way to communicate their meaning to the user?